README.md

```diff
@@ -29,6 +29,37 @@ To do this, do the following:
 - From the `stencil-library` directory, run `yarn link`
 - From the `react-library` run `yarn link "@justifi/webcomponents"`
 
+## Lerna Management
+
+This project now uses [Lerna](https://lerna.js.org/) to manage the versions of the multiple packages. Here are some common Lerna commands you might need:
+
+### Bootstrapping
+
+Lerna will ensure dependencies are linked together:
+
+```bash
+lerna bootstrap
```

Comment:
It looks like `bootstrap` is deprecated in v7 and higher. Do we want to use this command? I think it's replaced by the workspaces you have below https://lerna.js.org/docs/legacy-package-management#migrating-from-lerna-bootstrap-lerna-add-and-lerna-link-in-lerna-v7-and-later